games-strategy/warmux/warmux-11.04.1-r2.ebuild

```diff
@@ -52,15 +57,14 @@ src_configure() {
 		--with-localedir-name="${EPREFIX}"/usr/share/locale \
 		--with-datadir-name="${EPREFIX}"/usr/share/${PN} \
 		--with-font-path="${EPREFIX}"/usr/share/fonts/dejavu/DejaVuSans.ttf \
+		--enable-nls \
 		$(use_enable debug) \
-		$(use_enable nls) \
 		$(use_enable unicode fribidi)
 }
 
 src_install() {
 	default
 
-	rm -f "${ED%/}"/usr/share/${PN}/font/DejaVuSans.ttf || die
```

Comment:
I know you're removing it because it's not installed, but it tips off the ebuild overlooked the change because of this -f.

It's installing 3 other fonts, and it wouldn't use dejavu if not for --with-font-path (we're essentially forcing a different font and installing a bundled one for nothing).

Could unbundle them if you want (imagine the two other non-font-path fonts would need workarounds), but I'm not picky about unbundling for fonts (not like we're talking outdated libraries here), so could alternatively just remove the dejavu dep along with --with-font-path, and add the relevant LICENSE for these fonts `|| ( Apache-2.0 GPL-3 ) UbuntuFontLicense-1.0 vlgothic`

Reply:
Where did you find license for these fonts?

Reply:
You can get them from their ebuilds:
media-fonts/wqy-microhei (although it's license is slightly wrong, it's one or the other `|| (  )`)
media-fonts/ubuntu-font-family
media-fonts/vlgothic (this has extra licenses but they don't apply to the used font as far as I can tell)